entries.py

```diff
@@ -73,7 +11,7 @@ def generate_plaintext(self):
 	pronunciation="", 
 	seealso="")
 
-BDC = Gloss("BDC", 
+BDC = GlossEntry("BDC", 
 	acronym_full="Biodata Catalyst", 
 	definition="", 
 	furtherreading="", 
```

Comment:
Since you provided default values in the constructor, you can get away with leaving out the empty entries in your GlossEntry objects here, for example:
```
BDC = GlossEntry("BDC",
                 acronym_full="Biodata Catalyst",
                 internal=False,
                 seealso="Biodata Catalyst")
```

This should cut out some lines of code from your definitions.

Reply:
I make new entries by copy-pasting and usually don't bother to delete the empty lines. But none of them use `internal` so I can at least delete those ones.